src/config/index.js

```diff
@@ -37,7 +37,7 @@ const config = {
   ],
   refinements: [
     {
-      type: 'hierarchical',
+      type: 'hierarchicall',
```

Comment:
Undo before merging.

Reply:
🤦‍♀️  I totally thought I removed it. Thanks for noticing